### PR TITLE
fix explicitly `undefined` values not falling back to default @ `InsertQueryBuilder.values`.

### LINKED ISSUE #398: SQL parser bug
First of all, thank you for writing this lightweight TS safe library. We're migrating our website from MongoDB to PostgreSQL. We tried Prisma and it was a hot pile of disaster in terms on performance. 

However, while setting up a few scripts, we found a bug inside this library. Our hands our currently full otherwise would have loved to jump into source and send a PR fix too. Here's how you can reproduce the bug: 

1. Go to https://codedamn.com/playground/4rv-jRRmU-BdpcEtjKm-0
2. Click on "Run code" button on top right
3. Go through the comments in the codebase

### PR DESCRIPTION
**Pull request by igalklebanov**
closes #398.

- opened multi-row inserts with defaults test cases to mysql as well.
- also removes some unused imports.

### CODE REVIEW

File: src/parser/insert-values-parser.ts [hunk 1]
```diff
@@ -91,34 +91,39 @@ function parseRowValues(
...
-    const defaultValue = DefaultInsertValueNode.create()
+  if (hasMissingColumns || hasUndefinedOrComplexColumns) {
+    let defaultValueNode: DefaultInsertValueNode
 
     return ValueListNode.create(
-      rowValues.map((it) =>
-        isUndefined(it) ? defaultValue : parseValueExpression(it)
+      rowValues.map((value) =>
+        isUndefined(value)
+          ? (defaultValueNode =
```
Comment (koskimas) [inline src/parser/insert-values-parser.ts:124]: Move the assignment away from the ternary. Do we really need this optimization? I don't think the old way caused any performance problems. We could even create a new instance each time and ditch the `defaultValueNode` variable.

File: src/parser/insert-values-parser.ts [hunk 2]
```diff
@@ -91,34 +91,39 @@ function parseRowValues(
...
 
-  if (columnMissing || complexColumn) {
-    const defaultValue = DefaultInsertValueNode.create()
+  if (hasMissingColumns || hasUndefinedOrComplexColumns) {
+    let defaultValueNode: DefaultInsertValueNode
 
     return ValueListNode.create(
-      rowValues.map((it) =>
-        isUndefined(it) ? defaultValue : parseValueExpression(it)
+      rowValues.map((value) =>
```
Comment (koskimas) [inline src/parser/insert-values-parser.ts:122]: Quite a lot of opinionated changes style changes. You'll have a lot of work ahead of you if you plan to get rid of all `it` variables. It has been used has the shorthand for `iterator` pretty much everywhere.

**Review by koskimas — APPROVED** [review]

File: src/parser/insert-values-parser.ts (hunk 2 above)
Comment (igalklebanov) [reply src/parser/insert-values-parser.ts]: Sorry, habits. 😢

File: src/parser/insert-values-parser.ts (hunk 1 above)
Comment (igalklebanov) [reply src/parser/insert-values-parser.ts]: I'll revert to the old way.

**Pull request merged**